File: ComputerBasics/Programs/Python/MultiArray.py
# Example of a 2D list (matrix)
matrix = [
    [1, 2, 3],
    [4, 5, 6],
    [7, 8, 9]
]

# Printing with nested loops
print("Printing a 2D list:")
for row in matrix:
    for element in row:
        print(element, end=" ") # Print elements of a row on the same line
    print() # Move to the next line after each row

File: ComputerBasics/Programs/java/MultiArray.java
public class MultiArray {
    public static void main(String[] args) {
        int[][] matrix = {
            {1, 2, 3, 10},
            {4, 5, 6, 11},
            {7, 8, 9, 12}
        };

        for (int i = 0; i < matrix.length; i++) {
            for (int j = 0; j < matrix[i].length; j++) {
                System.out.printf("%4d", matrix[i][j]); // Adjust %4d for desired spacing
            }
            System.out.println(); // New line after each row
        }
    }
}